proposals/i18n-getOSLanguage.md

```diff
@@ -0,0 +1,78 @@
+# Proposal: i18n.getOSLanguage>
+
+**Summary**
+
+Allow to get the language of your operating system as an [BCP47 language tag](https://www.w3.org/International/core/langtags/rfc3066bis.html).
```

Comment:
Given the discussion starting from https://github.com/w3c/webextensions/issues/252#issuecomment-2027025304, should we rename the proposal to `i18n.getOSLanguages` and let it return an array?

And also, guarantee the array to be non-empty?